cmd/ui/src/views/Explore/ExploreSearch/CypherSearch.tsx

```diff
@@ -159,78 +167,80 @@ const CypherSearch = () => {
 
     return (
         <>
-            <Box display={'flex'} gap={1}>
-                <Button
-                    className={`${classes.button} ${classes.iconButton}`}
-                    variant={'secondary'}
-                    onClick={() => {
-                        setShowCommonQueries((v) => !v);
-                    }}
-                    aria-label='Show/Hide Saved Queries'>
-                    <FontAwesomeIcon icon={faFolderOpen} />
-                </Button>
-
-                <div onClick={setFocusOnCypherEditor} style={{ flex: 1 }} role='textbox'>
-                    <CypherEditor
-                        className={darkMode ? classes.cypherEditorDark : classes.cypherEditor}
-                        value={cypherQuery}
-                        onValueChanged={(val: string) => {
-                            setCypherQuery(val);
-                        }}
-                        theme={darkMode ? 'dark' : 'light'}
-                        onKeyDown={(e: any) => {
-                            // if enter and shift key is pressed, execute cypher search
-                            if (e.key === 'Enter' && e.shiftKey) {
-                                e.preventDefault();
-                                handleCypherSearch();
-                            }
+            <Box display='flex' flexDirection='column' height='100%' overflow='hidden'>
```

Comment:
Not a huge deal but the overflow hidden attribute is making the buttons cut off in light mode. Doesnt seem like we need it though, but I will defer to you. 

Here is a screenshot with the overflow=hidden as implemented and below it one without the attribute:

<img width="1202" alt="Screenshot 2024-09-10 at 4 18 13 PM" src="https://github.com/user-attachments/assets/3a8b0d57-0a6b-4cba-bce9-5d69126378f6">
<img width="749" alt="Screenshot 2024-09-10 at 4 18 51 PM" src="https://github.com/user-attachments/assets/64ecc280-d205-4c12-ab92-9c0de068289b">